src/vellum/workflows/nodes/core/templating_node/tests/test_templating_node.py

```diff
@@ -74,7 +74,7 @@ class BoolTemplateNode(TemplatingNode[BaseState, bool]):
 
 def test_templating_node__json_output():
     # GIVEN a templating node that outputs JSON
-    class JSONTemplateNode(TemplatingNode[BaseState, Any]):
+    class JSONTemplateNode(TemplatingNode[BaseState, Json]):
```

Comment:
[blocking] Can we add a test to confirm that we're able to produce the expected JSON representation when we serialize a node that looks like this?